renovate.json

```
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "config:base"
```

Some additional settings I recommend:

```suggestion
    "config:base",
    ":preserveSemverRanges",
    ":ignoreUnstable",
    ":respectLatest"
```